rcgen/src/error.rs

```diff
@@ -13,6 +13,16 @@ pub enum Error {
 	#[cfg(feature = "x509-parser")]
 	/// Invalid subject alternative name type
 	InvalidNameType,
+	/// Invalid PrintableString type
+	InvalidPrintableString,
```

Comment:
I think this should be just one variant, optionally with an enum to distinguish the string type.

Reply:
I've made it into one variant with an enum to distinguish the string type. I've also added some context to the error